source/Government.cpp

```diff
@@ -116,6 +118,8 @@ void Government::Load(const DataNode &node)
 			bribe = child.Value(1);
 		else if(child.Token(0) == "fine" && child.Size() >= 2)
 			fine = child.Value(1);
+		else if(child.Token(0) == "bribe factor" && child.Size() >= 2)
```

Comment:
I would expect to also see a `"reputation hit"` factor when the player flees. (Not a large factor for most governments, but fleeing should have a little bit longer-term effects.)